api.md

```diff
@@ -252,7 +253,7 @@ data node by:
 2. Creating the database given in `database` that will serve as the
    new data node.
 3. Loading the TimescaleDB extension in the new database.
-4. Setting metadata to make the data node part of the distributed
```

Comment:
I don't think the removing `the` is appropriate, but I am not an English native person.

Reply:
One of my more common typos. Thanks for catching it.

Reply:
Yeah, think `the` should be there.